app/views/devise/registrations/new.html.slim

```diff
@@ -1,81 +1,74 @@
-                  span.input-group-text
-                    i.fas.fa-phone
-                .field
-                  = f.text_field :mobile,
-                                  placeholder: 'mobile phone',
-                                  autocomplete: 'mobile'
-
-              .input-group.form-group
-                .input-group-prepend
-                  span.input-group-text
-                    i.fas.fa-key
-                .field
-                  = f.password_field :password,
-                                     placeholder: 'password',
-                                     autocomplete: 'new-password'
-
-              .input-group.form-group
-                .input-group-prepend
-                  span.input-group-text
-                    i.fas.fa-key
-                .field
-                  = f.password_field :password_confirmation,
-                               placeholder: 'password confirmation',
-                               autocomplete: 'new-password'
-
-              .actions
-                = f.submit 'Sign Up'
+      .content-abc.mx-auto.shadow.m-3
+        h1.pt-4.font-weight-bold.row.justify-content-center
+          = title t('devise.registrations.new.tab_title'),
+            t('devise.registrations.new.registration_title')
+        hr
+        = simple_form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f|
+          = f.error_notification
+          .container.form-prop
+            .container.form-horizontal.mb-2.mt-4
+              p.form-group
+                .row
+                  i.fas.fa-user.fa-lg.fa-fw.mt-2.control-label.col-sm-2.icon-prop
+                  .col-sm-10
+                    = f.input :first_name, autofocus: true, label: false,
+                      placeholder: t('devise.registrations.new.f_name_placeholder'),
+                      input_html: { class: 'form-control input-prop' }
+              p.form-group
+                .row
+                  i.fas.fa-user.fa-lg.fa-fw.mt-2.control-label.col-sm-2.icon-prop
+                  .col-sm-10
+                    = f.input :last_name, autofocus: true, label: false,
+                      placeholder: t('devise.registrations.new.l_name_placeholder'),
+                      input_html: { class: 'form-control input-prop' }
+              p.form-group
+                .row
+                  i.fas.fa-envelope.fa-lg.fa-fw.mt-2.control-label.col-sm-2.icon-prop
+                  .col-sm-10
+                    = f.input :email, autofocus: true, label: false,
+                      placeholder: t('devise.registrations.new.email_placeholder'),
+                      input_html: { class: 'form-control input-prop' }
+              p.form-group
+                .row
+                  i.fas.fa-venus-mars.fa-lg.fa-fw.mt-2.control-label.col-sm-2.icon-prop
+                  .col-sm-10
+                    = f.input :sex, collection: [[t('devise.registrations.new.sex.male'), 'male'],
```

Comment:
there is a helper which creates options, please look at UsersHelper

Reply:
Ohhh, I'm realy forgot about UserHelper